src/main/webapp/WEB-INF/foodmeeting/searchImage.jsp

```diff
@@ -0,0 +1,39 @@
+<%@ page contentType="text/html" pageEncoding="UTF-8" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib prefix="t" tagdir="/WEB-INF/tags" %>
+
+<div class="modal-content" style="height:65%">
+	<div class="image-links">
+		<c:forEach var="image" items="#{images}">
+			<div class="meeting grid-item"
```

Comment:
Let's dont' use the same styles that we use for the food meeting page. If we change those styles for some reason, the images masonry layout would be affected.  We can create another style sheet file for the searchImage styles.